kolibri_instant_schools_plugin/assets/src/views/vf-accent-button/index.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+
+  <k-button
```

Comment:
@indirectlylit how's this look?

also, hello from 39,000 feet

Reply:
Looking good! ☁️  ✈️